src/docs/common/snippets/common-graal-with-plugins-multi.adoc

```diff
@@ -53,6 +53,28 @@ To generate native executables for each application using Maven run:
 
 The native executable is created in the `target` directory and can be run with `target/micronautguide`.
 
+It is possible to customize the name of the native executable or pass additional build arguments using the https://graalvm.github.io/native-build-tools/latest/maven-plugin.html[Maven plugin for GraalVM Native Image building]. Declare the plugin as following:
+
+.pom.xml
+[source,xml]
+----
+<plugin>
+    <groupId>org.graalvm.buildtools</groupId>
+    <artifactId>native-maven-plugin</artifactId>
+    <version>0.10.3</version>
```

Comment:
Maybe we shouldn't hardcode the plugin version and use `${native-maven-plugin.version}`